send-transaction-service/src/send_transaction_service.rs

```diff
@@ -115,6 +115,8 @@ pub struct Config {
     pub batch_send_rate_ms: u64,
     /// When the retry pool exceeds this max size, new transactions are dropped after their first broadcast attempt
     pub retry_pool_max_size: usize,
+    pub tpu_peers: Option<Vec<SocketAddr>>,
```

Comment:
What's the value of an Option here, instead of just a Vec that may be empty (when flag isn't set)?

Reply:
pedantry mostly. preference for type safety of implicit sentinels